server/repository/src/db_diesel/message_row.rs

```diff
@@ -0,0 +1,174 @@
+use super::{
+    store_row::store, ChangeLogInsertRow, ChangelogRepository, ChangelogTableName, RowActionType,
+    StorageConnection,
+};
+use crate::{RepositoryError, Upsert};
+
+use chrono::NaiveDateTime;
+use diesel::prelude::*;
+use diesel_derive_enum::DbEnum;
+use serde::{Deserialize, Serialize};
+
+#[derive(DbEnum, Debug, Clone, PartialEq, Eq, Hash, Default, Serialize, Deserialize)]
+#[DbValueStyle = "SCREAMING_SNAKE_CASE"]
+#[PgType = "message_status"]
+pub enum MessageRowStatus {
+    #[default]
+    New,
+    Processed,
+    Error,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Default, Serialize, Deserialize)]
+pub enum MessageRowType {
+    #[default]
+    RequestFieldChange,
+    #[serde(untagged)]
+    Other(String),
+}
+
+impl From<String> for MessageRowType {
+    fn from(value: String) -> Self {
+        serde_json::from_str(&value).unwrap_or_default()
+    }
+}
+
+impl From<MessageRowType> for String {
+    fn from(value: MessageRowType) -> Self {
+        serde_json::to_string(&value).unwrap_or_default()
+    }
+}
+
+table! {
+    message (id) {
```

Comment:
```suggestion
    sync_message (id) {
```

Can we can it sync_message or similar, feels like it could be a confusing if we add some other kind of messaging in the future?